build/build-validation.js

```diff
@@ -24,7 +24,7 @@ module.exports.defaultInitOptions = ${JSON.stringify(defaultInitOptions)}
 const defaultInitOptions = {
   connectionTimeout: 0, // 0 sec
   keepAliveTimeout: 72000, // 72 seconds
-  forceCloseConnections: false, // keep-alive connections
+  forceCloseConnections: true, // keep-alive connections
```

Comment:
This is dangerous. There are ramifications that the user must know and account for. I disagree with enabling this by default.

Reply:
I can easily revert this. But enabling this will IMHO make close behave a bit more like someone might expect.
Which ramification are you talking about? I can definitely update the docs if needed.

Reply:
What ramifications? Those sockets are _idle_, there is no upstanding requests (minus the in-flight that we have not received yet).

On the other hand, making this the default improves the development experience and matches what people expects.

Reply:
I vote for not changing the `default`.
Add new option for closing behavior as `idle`, and when `forceCloseConnections` is `true`. It will be equal to `all` (which means call `closeAllConnections`).

Reply:
Note that `forceCloseConnections` is not equal to `all` right now, only `idle`.

Reply:
If in this pr, `forceCloseConnections` is equal to `all`, I think it's better to use `false` by default

Reply:
I agree.